cabal.project

```diff
@@ -165,6 +165,8 @@ constraints:
   -- ones they reuse the one from 'some', but there isn't e.g. a proper version
   -- constraint from dependent-sum-template (which is the library we actually use).
   , dependent-sum > 0.6.2.0
+  -- needed until we update past cardano-ledger@43f7c7318e38c501c2d2a2c680251c7c1f78d0fd
+  , hashable < 1.3.4.0
```

Comment:
If this PR goes after https://github.com/input-output-hk/ouroboros-network/pull/3530 we can drop this change.